### DISCUSSION TITLE
System fallback policy

### DISCUSSION THREAD

**Discussion by kyoushinosensei** [mashb1t/Fooocus#50] [Q&A] [answered]
Hi.)) Please, tell me - does the System fallback policy in the nVidia control panel somehow affect the operation? RTX 2080 Ti graphics card (11 GB), DDR4 2133 mHz 32 GB RAM

**Comment by mashb1t** [comment, marked as answer]
yes, most likely, but swap settings from troubleshooting guide are as important if not even more.

see https://nvidia.custhelp.com/app/answers/detail/a_id/5490/~/system-memory-fallback-for-stable-diffusion & https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md
